2.Installation/2.3.InstallUsingComposer.md

```diff
@@ -0,0 +1,16 @@
+2.1 Installing using Composer
+=============================
+ 
+Installing Fluid Powered TYPO3 with Composer is a simple command-line operation:
+
+Prerequisites: 
+- You have to use TYPO3 in Composer-mode. Please look at https://composer.typo3.org/ for more information. 
+- It is recommended that you have added `https://composer.typo3.org/` to your list of repositories.

```

Comment:
This is actually *not* recommended anymore. TYPO3 itself is also moving away from it. FluidTYPO3 is fully on packagist including all dependencies, so it is only in the case you need TER extensions which are only on TER and you do not want to add the "vcs" repository that you would need that composer repository.